src/app/f2f/controllers/landingPage.js

```diff
@@ -30,6 +31,7 @@ class LandingPageController extends BaseController {
 			return data;
 		} catch (error) {
 			console.log("Error calling /sessionConfiguration");
```

Comment:
Why do we have cosole.log?

Reply:
Being used [here](https://github.com/alphagov/di-ipv-cri-f2f-front/blob/main/src/app/f2f/controllers/landingPage.test.js#L51)